public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js

```diff
@@ -150,6 +156,12 @@ export default class AlertsDashboardFlyoutComponent extends Component {
     }
   };
 
+  getLocalClusterName = async () => {
```

Comment:
Nit: Not a blocker but would rename this to something like `setLocalClusterName` since we are not returning anything in here